### ISSUE TITLE
DynamicProcessor should not validate

### ISSUE THREAD

**Issue by lejard-h** [Jaguar-dart/jaguar_serializer#143]
Dynamic processor should let pass everything

Iterable, DateTime, Object ...

The goal is to tell the serializer to not process the field.

**Comment by lejard-h** [comment]
same for `const Field(dontEncode: true, dontDecode: true)`;

this should work.

```dart
final map = { 'date': new DateTime() };

final obj = serializer.fromMap(map);
```

**Comment by tejainece** [comment]
`dontEncode` and `dontDecode` mean ignore while encoding and ignore while decoding, respectively. They don't mean "just pass as it is".

**Comment by tejainece** [comment]
Should there be a `PassAllProcessor`?

`DynamicProcessor` is processes `dynamic` by default, if no processor is set on it. It is supposed to only parse valid fields.

**Comment by tejainece** [comment]
Please reopen, if it is still an issue.

**Comment by lejard-h** [comment]
Yes it is an issue, for example, firebase provide partially decoded object with Datetime, like the example I provide

**Comment by tejainece** [comment]
Would a `PassAllProcessor` fit the need?

**Comment by tejainece** [comment]
Would you like to write the pull request?

I don't use firebase. It is probably easier for you to test that it works with firebase as well.

**Comment by lejard-h** [comment]
It won't be possible sorry, passing my test below is enough, it should work with any object

**Comment by tejainece** [comment]
Fixed by https://github.com/Jaguar-dart/jaguar_serializer/pull/145/commits/57ac08f53a822f5ba0d9a4a0812eb68d75037960

**Comment by tejainece** [comment]
Use `PassProcessor` by default for `dynamic` fields
